easybuild/easyconfigs/c/Clang/Clang-3.8.1-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
+# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
+# Authors:: Ward Poelmans <wpoely86@gmail.com>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = "Clang"
+version = "3.8.1"
+
+homepage = "http://clang.llvm.org/"
+description = """C, C++, Objective-C compiler, based on LLVM.  Does not
+ include C++ standard library -- use libstdc++ from GCC."""
+
+# Clang also depends on libstdc++ during runtime, but this dependency is
+# already specified as the toolchain.
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+# Do not set optarch to True: it will cause the build to fail
+toolchainopts = {'optarch': False}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = [
+    "llvm-%(version)s.src.tar.xz",
+    "cfe-%(version)s.src.tar.xz",
+    "compiler-rt-%(version)s.src.tar.xz",
+    "polly-%(version)s.src.tar.xz",
+    "openmp-%(version)s.src.tar.xz",
+]
+
+dependencies = [
+    ('GMP', '6.1.1'),
+#    ('ISL', '0.17.1'),
+]
+
+builddependencies = [
+    ('CMake', '3.6.1', '', ('GCCcore', '5.4.0')),
```

Comment:
Hardcoding the subtoolchain isn't sane either imho, since it conflicts with using `--try-toolchain`.

We have lots of easyconfigs for CMake, indeed, but this should be handled with in another way (i.e. with fat `.yeb` easyconfigs).